src/test/java/se/sundsvall/byggrintegrator/integration/byggr/ByggrIntegrationCacheTest.java

```diff
@@ -29,16 +27,14 @@
 import generated.se.sundsvall.arendeexport.GetRoller;
 import generated.se.sundsvall.arendeexport.GetRollerResponse;
 import generated.se.sundsvall.arendeexport.Roll;
+import se.sundsvall.byggrintegrator.Application;
 
 @ExtendWith(MockitoExtension.class)
```

Comment:
Do you need this in a SpringBootTest?

Reply:
No, I missed that one when adjusting the test class, will remove.